### DISCUSSION TITLE
Bigger bottom informations

### DISCUSSION THREAD

**Discussion by tangsgod** [organicmaps/organicmaps#497] [Ideas]
First of, thank you VERY much for making this app. As I'm concerned about privacy, it seems to be the best option on Android now.
This is my first message on github ever but I wanted to share my opinion on this great growing app :)

So, my suggestion is to make the information area at the bottom of the screen a bit bigger.
I'm talking about here:
![Screenshot_20210605-131157_Organic_Maps~2](https://user-images.githubusercontent.com/85394837/120890256-b2163980-c601-11eb-8414-19116f3c5615.png)
![Screenshot_20210605-131322_Organic_Maps~2](https://user-images.githubusercontent.com/85394837/120890257-b2aed000-c601-11eb-8b37-d3323d83ec8b.png)
![Screenshot_20210605-131422_Organic_Maps~2](https://user-images.githubusercontent.com/85394837/120890258-b3476680-c601-11eb-8162-3ed5407d905b.png)

I'm on a Redmi Note 8 (6.3 inches) and those infos are too little to be read quickly and easly, specially while driving. (And yes, I have a good view :) )

Thank you to consider this request, I may propose some more after using it for a while.

Tangsgod

**Comment by GLLM** [comment]
Could this be a setting with the following options : smaller / small / normal / big / bigger ?

**Reply by tangsgod** [reply]
Yes, of course it could be like this :)

**Comment by biodranik** [comment]
It's a good idea to increase the size proportionally to the screen size in navigation mode. We'll see what we can do. Thanks!
